scripts/broken-link.mjs

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+
+/**
+ * Script to check for broken links using lychee in a Docker container
+ * Usage:
+ *   npm run broken-link                     # Checks ./content/ directory
+ *   npm run broken-link terraform-plugin-framework  # Checks specific directory
+ *   npm run broken-link dir1 dir2 dir3      # Checks multiple directories
+ */
+
+import { execSync } from 'child_process';
+import path from 'path';
```

Comment:
@im2nguyen let's remove this, as it doesn't look to be used.

Reply:
```suggestion
```